mlflow/models/model_config.py

```diff
@@ -3,21 +3,21 @@
 
 import yaml
 
-import mlflow
+__mlflow_model_config__ = None
 
 
+# TODO: Let ModelConfig take in a dictionary instead of a file path
 class ModelConfig:
     """
     ModelConfig used in code to read a YAML configuration file, and this configuration file can be
     overridden when logging a model.
     """
 
     def __init__(self, *, development_config: Optional[str] = None):
-        # TODO: Update global path after we pass in paths using model_config
-        _mlflow_rag_config_path = getattr(
-            mlflow.langchain._rag_utils, "__databricks_rag_config_path__", None
-        )
-        self.config_path = _mlflow_rag_config_path or development_config
+        config = globals().get("__mlflow_model_config__", None)
+        # backwards compatibility
+        rag_config = globals().get("__databricks_rag_config_path__", None)
```

Comment:
Where is this global set? 
Do we need to use `mlflow.langchain._rag_utils` or something?

Reply:
Yes, I think in previous versions they would set mlflow.langchain._rag_utils. But not sure if we need any backwards compatibility for the globals because you would be using the same mlflow version to set and get... 🤔